3 - Assignment/240326 - Rust Basic + Roguelike Game, Week 1/collatz-sequence/src/lib.rs

```diff
@@ -1,6 +1,17 @@
 /// Determine the length of the collatz sequence beginning at `n`.
 pub fn collatz_length(mut n: i32) -> u32 {
-    todo!("Implement this")
+    let mut length = 1;
+    
+    while n != 1 {
+        if n % 2 == 0 {
+            n /= 2;
+        } else { 
+            n = 3 * n + 1;
+        }
```

Comment:
이렇게 정리하면 좀 더 깔끔해집니다.
```suggestion
        n = if n % 2 == 0 { n / 2 } else { 3 * n + 1 };
```